indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskMaster.java

```
@@ -416,4 +418,22 @@ public Map<String, Long> getBlacklistedTaskSlotCount()
       return null;
     }
   }
+
+  /**
+   * Process the results of synchronization from storage
+   *
+   * @param syncResult to be processed
+   * @param taskStorage stores tasks
+   * @param taskRunner runs tasks and holds state
+   */
+  private void processSyncResult(SyncResult syncResult, TaskStorage taskStorage, TaskRunner taskRunner)
+  {
+    String taskReacquisitionFailure = "Failed to reacquire lock";
+    for (Task task : syncResult.getTasksToFail()) {
+      // Shutdown task if it is running
+      taskRunner.shutdown(task.getId(), taskReacquisitionFailure);
+      // Mark as failed
+      taskStorage.setStatus(TaskStatus.failure(task.getId(), taskReacquisitionFailure));
```

should we ignore any exception in this block?